core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.scss

```diff
@@ -0,0 +1,37 @@
+@use "variables" as *;
+
+form {
+    display: flex;
+    flex-direction: column;
+    padding-left: 30px;
+
+    .p-input-icon-right {
+        position: relative;
+        width: 100%;
+
+        input {
+            width: 100%;
+            box-sizing: border-box;
+            border: 1px solid $color-palette-gray-700;
+            border-radius: 5px;
+            outline: none;
+            color: $color-palette-gray-700;
+            font-size: 16px;
+        }
+
+        button {
+            display: flex;
+            align-items: center;
+            position: absolute;
+            right: 0;
+            top: 0;
+            bottom: 0;
+            background: none;
+            border: none;
+            outline: none;
+            cursor: pointer;
+            padding-right: 15px;
+            color: $color-palette-primary;
+        }
```

Comment:
After using a primeNG button, you probably won't need this. It has its own styles from the theme.

Reply:
primeNG button is not replacing the style of this one, it's totaly off, but I will remove some propeties from here